### ISSUE TITLE
Installation Issue

### ISSUE THREAD

**Issue by Hax108** [5T33Z0/Lenovo-T530-Hackintosh-OpenCore#40]
i get this error when booting into ventura from fresh install 
![IMG_0075](https://user-images.githubusercontent.com/131800236/234406383-88af09a7-f18c-4aa4-83d4-9903a0ec9c45.JPG)

**Comment by 5T33Z0** [comment]
Did you follow the install instructions for Ventura? 
https://github.com/5T33Z0/Lenovo-T530-Hackintosh-OpenCore/blob/main/macOS_Install/OpenCore/OC_macOS13.md

**Comment by Hax108** [comment]
> Did you follow the install instructions for Ventura? 
> https://github.com/5T33Z0/Lenovo-T530-Hackintosh-OpenCore/blob/main/macOS_Install/OpenCore/OC_macOS13.md
> 

yes i did created the installer with opencore legacy patcher my t530 has the same specs

**Comment by 5T33Z0** [comment]
But I can see macOS running in the screenshot. Or is this is a differnt installation?

BTW: you can disable the camera in BIOS. No need to put cover the lens.

**Comment by Hax108** [comment]
> But I can see macOS running in the screenshot. Or is this is a differnt installation?
> 
> BTW: you can disable the camera in BIOS. No need to put cover the lens. 

i have monterey installed on my msata. i tried installing ventura to an external hard drive via usb and get that error in verbose mode when i load up monterey the error i got from the ventura installer shows up

**Comment by 5T33Z0** [comment]
Okay. Try installing on a new internal volume instead. Enable the `AppleCpuPmCfgLock` Qurik if it's not enabled already.

**Comment by Hax108** [comment]
it worked im finishing the rest of the process.
